README.md

```diff
@@ -1,24 +1,163 @@
-# Sensu Go Remediation Handler
+# Sensu Remediation Handler
 
-The Sensu Go Remediation Handler is a simple [Sensu Go event handler][handler]
-for configuring self-healing workflows using Sensu. This handler was heavily
-inspired by [Nick Stielau's "Sensu Remediator"][remediator] (circa 2012).
+[![Bonsai Asset Badge](https://img.shields.io/badge/Bonsai-Download%20Me-brightgreen.svg?colorB=89C967&logo=sensu)](https://bonsai.sensu.io/assets/sensu/sensu-remediation-handler) [![Build Status](https://travis-ci.com/sensu/sensu-remediation-handler.svg?branch=master)](https://travis-ci.com/sensu/sensu-remediation-handler)
 
-The Sensu Go Remediation Handler &ndash; and other similar "self healing"
-workflows in Sensu &ndash; use a few simple Sensu features:
+- [Overview](#overview)
+- [Configuration](#configuration)
+  - [Environment Variables](#environment-variables)
+  - [Annotations](#annotations)
+    - [Annotation specification](#annotation-specification)
+    - [Remediation action specification](#remediation-action-specification)
+- [Setup](#setup)
+- [Usage Example](#usage-example)
+  - [Example "Unscheduled" Check (Remediation Action)](#example--unscheduled--check--remediation-action-)
+  - [Example Check Definition and Remediation Request Configuration](#example-check-definition-and-remediation-request-configuration)
+- [Acknowledgements](#acknowledgements)
+
+## Overview
+
+The Sensu Remediation Handler is a [Sensu Go event handler][handler]
```

Comment:
Is there a reason to use words for relative links rather than numbers? In the docs I think it's mainly numbers, but every once in a while there will be a word instead.

Reply:
The only advantage I see for words over numbers is that I often feel like i need to reorder numbers when adding new links above existing links. This is tedious but usually not necessary.

I think consistency matters most. The style guide for sensu-docs doesn't dictate which link style to use, but maybe it should.

Reply:
I agree, the style guide should cover this. I like numbers b/c I think it looks more organized and it doesn't require you to spend time deciding on a word/phrase to use. But I also feel compelled to keep them in order. Maybe we could standardize with "use numbers; no need to cite them in order."

Reply:
https://github.com/sensu/sensu-docs/wiki/Sensu-docs-style-guide#links